docker/complete/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM mesosphere/mesos:0.22.0-1.0.ubuntu1404
+MAINTAINER Mesosphere <support@mesosphere.io>
+
+RUN locale-gen en_US.UTF-8
+RUN dpkg-reconfigure locales
+ENV LANG en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+
```

Comment:
My goal is to avoid packages trying to ask the user questions. See http://www.microhowto.info/howto/perform_an_unattended_installation_of_a_debian_package.html